source/process/gitlab-process.rst

```diff
@@ -29,10 +29,15 @@ If the release contains a security update, PM owner emails @marin and @briann in
 Testing
 ----------------
 
-The following steps are taken to test the Mattermost package in GitLab Omnibus:
+The following steps are taken to test the Mattermost package before each merge request to GitLab Omnibus:
 
-1. Each Mattermost version is tested on a GitLab Omnibus build at `http://ci-linux-gitlab-omnibus.mattermost.com/ <http://ci-linux-gitlab-omnibus.mattermost.com/>`__. Testing covers all core Mattermost features, including notifications and GitLab SSO.
-2. Before each merge request to GitLab Omnibus, upgrade is tested `following the steps below <https://docs.mattermost.com/process/gitlab-process.html#testing-upgrade-process>`__, using the `nightly Omnibus packages <https://packages.gitlab.com/gitlab/nightly-builds>`__ to validate the integration. This is so the packaging code and OAuth setup can be tested as well, which has historically been the main source of issues. Other test areas include:
+1. Pull the latest master build of GitLab Omnibus, using the `nightly Omnibus packages <https://packages.gitlab.com/gitlab/nightly-builds>`_.
+2. Update the Mattermost version in `config/softwater/mattermost.rb`.
+3. Update the version in the GitLab documentation and add a changelog entry.
```

Comment:
Version of the documentation itself, or the version found in the documentation - and is it the GitLab version that's being updated?